script.js

```diff
@@ -11,31 +11,104 @@ function createOptionGames(selector) {
 createOptionGames('gameIn');
```

Comment:
Calling functions on the root is not the right way to do it. Try to call on html